platform/ios/src/MGLMapView.h

```diff
@@ -212,7 +212,7 @@ IB_DESIGNABLE
 /** 
  Currently active style classes, represented as an array of string identifiers.
  */
-@property (nonatomic) NS_ARRAY_OF(NSString *) *styleClasses;
+@property (nonatomic) NS_ARRAY_OF(NSString *) *styleClasses __attribute__((deprecated("Use -style.styleClasses:.")));
```

Comment:
Nit: the `-` is for selector names, not key paths. How about: `Use -[MGLStyle styleClasses] via -[MGLMapView style].` Or: `Use style.styleClasses.`